TodoListApp/TodoList.Api.Tests/Comparers/ItemComparer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using TodoList.Api.Models;
+
+namespace TodoList.Api.Tests.Comparers
+{
+    public class ItemComparer: IEqualityComparer<Item>
+    {
+        public bool Equals(Item x, Item y)
+        {
+            if (ReferenceEquals(x, y)) return true;
+            if (ReferenceEquals(x, null)) return false;
+            if (ReferenceEquals(y, null)) return false;
+            if (x.GetType() != y.GetType()) return false;
+            return x.Id == y.Id && string.Equals(x.Text, y.Text);
+        }
+
+        public int GetHashCode(Item obj)
+        {
+            unchecked
```

Comment:
Allegedly, it improves performance, because overflow-checking for arithmetic operations is suppressed and therefore code execution is faster. And it's fine if the value overflows in this case.
But yeah, I only used it because R# suggested it to be honest.

Reply:
Allegedly, code execution is faster when overflow-checking is suppressed. And it's fine if the value overflows in this case, because we don't really care how big or small the value of hashcode is, we just want it to be the same for the same objects.
But to be honest, I only used it because R# suggested it😃